actions-on-google.js

```diff
@@ -81,9 +85,21 @@ class ActionsOnGoogle {
     }
 
     setLocale(locale) {
+        if (!SUPPORTED_LOCALES.concat(Object.keys(FALLBACK_LOCALES)).includes(locale)) {
+            throw new Error(`Unsupported locale: ${locale}`);
+        }
         this.locale = locale;
+        i18n.setLocale(locale);
     }
 
+    i18n_(name, params) {
+        if (params) {
+            return i18n.__(name, params);
+        } else {
+            return i18n.__(name);
+        }
+    };
```

Comment:
Does this need a semicolon? I don't think it's part of the spec for a class-level function.

Reply:
@Fleker Oh sorry, it is unnecessary. I will remove the character.